addons/portal/controllers/portal.py

```diff
@@ -172,6 +172,11 @@ def home(self, **kw):
         values = self._prepare_portal_layout_values()
         return request.render("portal.portal_my_home", values)
 
+    @http.route('/web/login_successful', type='http', auth="user", website=True)
```

Comment:
I don't think you need to redefine the route here
```suggestion
```

Reply:
As we discussed, I do since it is not an override of the same class.